core/src/mindustry/core/NetClient.java

```diff
@@ -339,6 +340,13 @@ public static void infoToast(String message, float duration){
 
         ui.showInfoToast(message, duration);
     }
+    
+    @Remote(variants = Variant.both)
+    public static void warningToast(int unicode, String text){
+        if(text == null || Fonts.def.getData().getGlyph((char)unicode) == null) return;
```

Comment:
Wait, you should use `Font.icon` for this, as that's where the large icons are.

Reply:
like this?